bot/exts/discord/discord.py

```diff
@@ -94,14 +144,30 @@ async def on_embed_error(
             # It's required to do `disnake.HTTPException()`,
             # instead of just `HTTPException()`
             case disnake.HTTPException():
-                raise commands.BadArgument(
-                    "Invalid URL provided."
-                )
+                raise commands.BadArgument("Invalid URL provided.")
             case _:
                 raise commands.BadArgument(
                     "No valid embed could be generated from the given input."
                 )
-            
+
+    @server.error
+    async def server_error(
+        self, inter: ApplicationCommandInteraction, error: Exception
+    ):
+        if isinstance(error, ConversionError):
+            if isinstance(error.original, GuildNotFound):
+                embed = create_embed(
+                    embed_type="error",
+                    title=str(error.original),
+                    description="Invalid ID or the bot does not have access to the server.",
+                )
+            if isinstance(error.original, BadInviteArgument):
+                embed = create_embed(
+                    embed_type="error",
+                    description=str(error.original),
+                )
+            await inter.response.send_message(embed=embed, ephemeral=True)
+
```

Comment:
I'm thinking of making an error handler for catching all these errors, but that hasn't happened yet. This is pretty good to have until then.